src/Elastic.Apm/Api/Label.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to Elasticsearch B.V under
+// one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+namespace Elastic.Apm.Api
+{
+	public class Label
```

Comment:
Should add some XML documentation to indicate what this represents, and highlight the implicit converion from supported values

Reply:
Agreed - added XML doc.